Upstream-Status: Pending

This patch fixes following issue with automake 1.12

| configure.ac:42: error: automatic de-ANSI-fication support has been removed
...
| automake: warnings are treated as errors
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'liblightcomp.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/03

Index: lighttpd-1.4.30/configure.ac
===================================================================
--- lighttpd-1.4.30.orig/configure.ac
+++ lighttpd-1.4.30/configure.ac
@@ -24,6 +24,11 @@ dnl AC_PROG_CXX
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
 
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
+
 dnl check environment
 AC_AIX
 AC_ISC_POSIX
@@ -39,7 +44,6 @@ AM_CONDITIONAL(NO_RDYNAMIC, test x$NO_RD
 AC_EXEEXT
 
 dnl more automake stuff
-AM_C_PROTOTYPES
 
 dnl libtool
 AC_DISABLE_STATIC
